entrypoint.sh

```diff
@@ -182,10 +186,10 @@ then
 fi
 
 # set outputs
-echo "::set-output name=new_tag::$new"
-echo "::set-output name=part::$part"
-echo "::set-output name=tag::$new" # this needs to go in v2 is breaking change
-echo "::set-output name=old_tag::$tag"
+setOutput "new_tag" "$new"
+setOutput "part" "$part"
+setOutput "tag" "$new"
```

Comment:
can you please maintain the comment for v2 `# this needs to go in v2 is breaking change`

Reply:
Added this comment back in.  Sorry about that!